themes/cms-forms/templates/SilverStripe/Forms/ReadonlyField.ss

```diff
@@ -1,6 +1,10 @@
-<p id="$ID" tabIndex="0" class="form-control-static<% if $extraClass %> $extraClass<% end_if %>" <% include SilverStripe/Forms/AriaAttributes %>>
-	$FormattedValue
-</p>
+<p id="$ID"
+   tabindex="0"
+   class="form-control-static<% if $extraClass %> $extraClass<% end_if %>"
+   <% include SilverStripe/Forms/AriaAttributes %>
```

Comment:
SilverStripe/Forms/AriaAttributes will end up being one of these
https://github.com/silverstripe/silverstripe-admin/blob/3/templates/SilverStripe/Forms/Includes/AriaAttributes.ss
https://github.com/silverstripe/silverstripe-admin/blob/3/themes/cms-forms/templates/SilverStripe/Forms/Includes/AriaAttributes.ss

Seems like a confusing template pattern that's very inconsistently used, so I don't want to use it for this set of PRs

On the off chance that a project overrides these templates with something that does include either role or aria-readonly - the LAST html attributes in the element will have precedence

Reply:
I don't understand your hesitance to use a template that's very obviously intended for specifically this purpose but I'll put that aside, we can debate that another day.

However, given the last attributes will take precedence and we're considering that people may have overridden that template to include their own explicit role and area-readonly attribute, why are our attributes last?

Seems like the ones in the template should take precedent (i.e. be last) if someone has taken the time to override it to add these?